src/scripts/main.ts

```diff
@@ -28,15 +28,27 @@ function processRequest(r: WebRequest.OnBeforeRequestDetailsType): WebRequest.Bl
 
   // Get query text or empty string
   const url = new URL(r.url);
-  let queryText = url.searchParams.get('q')?.trim() ?? '';
+  let queryParam = url.searchParams.get('q');
+  if (queryParam === null) {
+    queryParam = url.searchParams.get('query');
+  } else if (queryParam === null) {
```

Comment:
After some brief testing it seems this should be

```suggestion
  }
  if (queryParam === null) {
```